go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/getlantern/utls
+
+go 1.16
+
+require (
+	github.com/andybalholm/brotli v1.0.4
+	github.com/klauspost/compress v1.13.6
```

Comment:
I needed the above dependencies (for Brotli and Zstandard compression respectively), so I added a go.mod file.